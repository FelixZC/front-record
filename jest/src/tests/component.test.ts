import { renderComponent } from '../examples/component';

test('renders the component correctly', () => {
  expect(renderComponent()).toMatchSnapshot();
});