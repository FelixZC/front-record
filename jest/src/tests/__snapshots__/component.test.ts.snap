// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component correctly 1`] = `"<div>Hello, World!</div>"`;
